columnflow/tasks/histograms.py

```diff
@@ -148,12 +148,16 @@ def run(self):
                 self.config_inst.get_variable(var_name)
                 for var_name in law.util.flatten(self.variable_tuples.values())
             )
-            for inp in (
+            for inp in ((
                 [variable_inst.expression]
                 if isinstance(variable_inst.expression, str)
                 # for variable_inst with custom expressions, read columns declared via aux key
                 else variable_inst.x("inputs", [])
-            )
+            ) + (
+                variable_inst.x("inputs", [])
```

Comment:
There is a case now where `variable_inst.x("inputs", [])` is used twice. Pretty sure this is not intentional.